src/main/java/Todo.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Class Todo to represent a Todo Task. A Todo Task
+ * consists of a description and a time.
+ */
+public class Todo extends Task {
+
+    /**
+     * Creates a new Todo Task with {@code description}.
+     *
+     * @param description the description/details of our task
+     */
+    public Todo(String description) {
+        super(description, "2099-12-31");

```

Comment:
Why does a todo have default time? Think of a better way to improve your design instead of creating objects with unnecessary information. hint: Think whether you need time attribute in Task class